styles.css

```diff
@@ -0,0 +1,284 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;500;600;700;800;900&display=swap');
+
+* {
+  padding: 0;
+  box-sizing: border-box;
+  margin: 0;
+  font-family: "Poppins", sans-serif;
+}
+
+body {
+  background-color: white;
+  position: static;
+}
+
+.portfolio-part-one {
+  box-sizing: border-box;
+  font-family: "Poppins", sans-serif;
+  background-position: 20px 10px;
+  background-image: url(./header-shapes\ mobile.png);
+  background-size: cover;
+  background-position-y: 40px;
+  background-position-x: 0;
+  background-repeat: no-repeat;
+  padding-bottom: 40px;
+}
+
+.container {
+  display: flex;
+  justify-content: space-between;
+  margin: 25px;
+}
+
+.logo-container a {
+  color: #6070ff;
+  font-weight: 700;
+  text-decoration: none;
+}
+
+.main-body {
+  margin: 25px;
+  margin-top: 60px;
+}
+
+.icons-title h3 {
+  font-size: 16px;
+  margin-top: 10px;
+  font-weight: 500;
+  color: #7f8cff;
+  line-height: 24px;
+}
+
+.paragraph h2 {
+  color: #172b4d;
+  margin-bottom: 15px;
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 52px;
+  margin-top: 20px;
+  padding-top: 28px;
+}
+
+p {
+  margin-bottom: 15px;
+  color: #344563;
+  font-size: 15px;
+  line-height: 24px;
+  line-height: 26px;
+}
+
+.paragraph p {
+  line-height: 24px;
+  font-size: 16px;
+  font-weight: 400;
+  color: #344563;
+}
+
+ul {
+  display: flex;
+}
+
+.pictures ul {
+  list-style: none;
+  display: flex;
+}
+
+ul li {
+  margin-top: 5px;
+  margin-bottom: 8px;
+  color: rgb(95, 95, 173);
+}
+
+.porfolio-list li {
+  margin-right: 25px;
+  font-size: 13px;
+  color: #7a869a;
+  font-weight: 600;
+}
+
+.pictures ul li {
+  margin-right: 10px;
+  margin-top: 10px;
+}
+
+i {
+  color: #505f79;
+  font-size: 20px;
+}
+
+.bars-container i {
+  color: #6070ff;
+}
+
+.portfolio .icons h5 {
+  font-size: 15px;
+  color: rgb(74, 141, 230);
+  font-weight: 500;
+}
+
+.button {
+  padding: 10px;
+  width: 180px;
+  font-size: 18px;
+  background-color: white;
+  border: 1px #396df2 solid;
+  border-radius: 8px;
+  color: #396df2;
+  cursor: pointer;
+  margin-top: 20px;
+}
+
+.button:hover {
+  color: black;
+  background-color: rgb(236, 127, 48);
+  border-color: chocolate;
+}
+
+.portfolio-two {
+  background-color: #e5e5e5;
+  height: 552px;
+  margin: 0 auto;
+  width: 100%;
+  position: static;
+  padding-top: 60px;
+}
+
+.portfolio-card {
+  display: grid;
+  background-color: #e5e5e5;
+  padding-left: 8px;
+  padding-top: 10px;
+  padding-right: 15px;
+  width: 375px;
+}
+
+.portfolio-card-about {
+  padding-bottom: 40px;
+}
+
+.portfolio {
+  padding: 20px;
+  background-color: white;
+  display: grid;
+  align-items: center;
+  align-content: center;
+  border-radius: 16px;
+  margin-bottom: 90px;
+  position: sticky;
+  width: 375px;
+}
```

Comment:
- [ ] Kindly use the width of 100% here this will make your about section look like the original about section template. check here :poin_down:

the template | your design 
---| ---
![image](https://user-images.githubusercontent.com/65068771/149622936-9c4de707-8f88-4a14-901c-f904953b3966.png) | ![image](https://user-images.githubusercontent.com/65068771/149622964-3e4d7c8a-f281-4f09-b771-10092e87506a.png)